pinot-broker/src/main/java/org/apache/pinot/broker/api/AccessControl.java

```diff
@@ -58,4 +58,19 @@ default boolean hasAccess(RequesterIdentity requesterIdentity) {
    * @return {@code true} if authorized, {@code false} otherwise
    */
   boolean hasAccess(RequesterIdentity requesterIdentity, Set<String> tables);
+
+  default boolean hasRBACAccess(RequesterIdentity requesterIdentity, String targetId,
+                                String targetType, String permission) {
+    return true;
+  }
+
+  /**
+   * If an API is neither annotated with RBACAuthorization nor ManualAuthorization,
+   * this method will be called to check the authorization.
+   * If the return is false, then API will be terminated by the filter.
+   * @return true to allow
+   */
+  default boolean defaultAuthorization(RequesterIdentity requesterIdentity) {
```

Comment:
Seems this is only associated with RBAC access. Can we make its name more specific